client/src/app/management/services/member-filter.service.ts

```diff
@@ -0,0 +1,84 @@
+import { MeetingRepositoryService } from 'app/core/repositories/management/meeting-repository.service';
+import { BaseFilterListService, OsFilter } from 'app/core/ui-services/base-filter-list.service';
+import { genders } from 'app/shared/models/users/user';
+import { ViewUser } from 'app/site/users/models/view-user';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class MemberFilterService extends BaseFilterListService<ViewUser> {
+    protected storageKey = 'MemberList';
+
+    private committeeFilterOptions: OsFilter<ViewUser> = {
+        property: 'committee_ids',
+        label: this.translate.instant('Committees'),
+        options: []
+    };
+
+    private meetingFilterOptions: OsFilter<ViewUser> = {
+        property: 'is_present_in_meeting_ids',
+        label: this.translate.instant('Meetings'),
+        options: []
+    };
+
+    public constructor(
+        store: StorageService,
+        historyService: HistoryService,
+        meetingRepo: MeetingRepositoryService,
+        committeeRepo: CommitteeRepositoryService,
+        private translate: TranslateService
+    ) {
+        super(store, historyService);
+
+        this.updateFilterForRepo(meetingRepo, this.meetingFilterOptions, this.translate.instant('No meeting'));
+        this.updateFilterForRepo(committeeRepo, this.committeeFilterOptions, this.translate.instant('No committee'));
+    }
+
+    protected getFilterDefinitions(): OsFilter<ViewUser>[] {
+        const staticFilterDefinitions: OsFilter<ViewUser>[] = [
+            {
+                property: 'is_active',
+                label: this.translate.instant('Active'),
+                options: [
+                    { condition: true, label: 'Is active' },
+                    { condition: false, label: this.translate.instant('Is not active') }
+                ]
+            },
+            {
+                property: 'isLastEmailSend',
+                label: this.translate.instant('Last email send'),
+                options: [
+                    { condition: true, label: this.translate.instant('Got an email') },
+                    { condition: false, label: this.translate.instant("Didn't get an email") }
+                ]
+            },
+            {
+                property: 'isVoteWeightOne',
+                label: this.translate.instant('Vote weight'),
+                options: [
+                    { condition: false, label: this.translate.instant('Has changed vote weight') },
+                    { condition: true, label: this.translate.instant('Has unchanged vote weight') }
+                ]
+            },
+            {
+                property: 'gender',
+                label: this.translate.instant('Gender'),
+                options: [
+                    ...genders.map(gender => ({ condition: gender, label: this.translate.instant(gender) })),
+                    { condition: '', label: this.translate.instant('No gender') }
+                ]
+            },
```

Comment:
Remove.
This cannot be filtered but only sorted. The server currently accepts any string for gender, for which filtering will not work

Reply:
Then we should adjust the server to accept only an enum of genders to achieve this "feature". @emanuelschuetze mentioned, that we should filter by gender.

Reply:
For the first time, I have removed that lines. When OpenSlides/openslides-backend#847 is resolved these filters can be adjusted accordingly.

Reply:
commenting them out and putting the issue in the comment would have been enough